oak-api/src/main/java/org/apache/jackrabbit/oak/api/jmx/QueryEngineSettingsMBean.java

```diff
@@ -139,5 +140,23 @@ void setQueryValidatorPattern(
 
     @Description("Get the query validator data as a JSON string.")
     String getQueryValidatorJson();
-
+    
+    
+    /**
+     * Set or remove java package/class names which are ignored from finding the 
+     * invoking class for queries.
+     * 
+     * it can be either Java package names or fully-qualified class names (package + class name).
+     * 
+     * @param classNames the class names to be ignored.
+     */
+    
+    @Description("Set or remove Java Package Name to ignore in Call Trace analysis")
+    void setIgnoredClassNamesinCallTrace(
+            @Description("package or fully qualified class names")
+            @Name("class names")
+            @NotNull String[] classNames);
+    
+    @Description("Get the Java package / fully qualified class names to ignore when finding the caller of query")
+    @NotNull String[] getIgnoredClassNamesinCallTrace();
```

Comment:
hi @joerghoh i looked at your patch again in my IDE not just the diff on github. and my IDE complains about a typo in this method name. it should be `getIgnoredClassNamesInCallTrace`